velox/exec/NestedLoopJoinProbe.cpp

```diff
@@ -357,9 +371,6 @@ bool NestedLoopJoinProbe::addToOutput() {
   // Check if the current probed row needs to be added as a mismatch (for left
   // and full outer joins).
   checkProbeMismatchRow();
-  if (output_ != nullptr) {
```

Comment:
we should probably add something in the method's documentation that is the caller's responsibility to resize the output now.

Reply:
It's updated.